### DISCUSSION TITLE
Omega and OneUI missing from available launchers

### DISCUSSION THREAD

**Discussion by Cris7006** [zixpo/candybar#146] [Q&A]
Hello,
It seems that Omega and OneUI are missing from the available launchers. They are supposedly available with the 3.17.4 update but it seems that "candybar-main" has not been updated to include these options. Since the available sample code is dependent on the candybar libraries, these options are not available. Please correct me if I am wrong as I am new to android and I apologise if I am wasting your time.

**Comment by Donnnno** [comment]
Omega is removed because it wasn't working anyways

**Reply by Cris7006** [reply]
Thank you. Is OneUI available? I am using a Samsing phone and I am unable to view my work for debugging

**Reply by Cris7006** [reply]
I guess I have asked the wrong question. Samsung id definitely the wrong phone to own

**Reply by Cris7006** [reply]
Anyone want a Samsung phone? I'm about to fold this one in half even though it's not foldable. I cannot change even one icon. Unbelievable. I guess android was handed over to AI. Now no one knows anything or has any control whatsoever

**Reply by moertel** [reply]
I contributed Samsung One UI to the list of supported launchers a few months ago: https://github.com/zixpo/candybar/pull/122

You need to install a little companion app called "Theme Park" but it's amazing and can even swap the icons on the lock screen and the edge panel.

**Reply by moertel** [reply]
@Cris7006 I'd actually be super curious to know where you looked for OneUI but couldn't find it. Did you check the latest CandyBar version? It should be in the list of supported launchers by default.
